### ISSUE TITLE
Global - Commoner head normal/shading issues

### ISSUE THREAD

**Issue by DarthParametric** [KOTORCommunityPatches/K1_Community_Patch#126]
N_TatComW_F, most notably used for Sharina outside the Anchorhead Hunting Lodge, has some weird shading issues on the lower edges of the eye sockets, the eyeball meshes clip through the eyelids and face, and the mouthbox and eyelid undersides have poor texture assignment.

**Comment by DarthParametric** [comment]
The problems are far more apparent in motion, but I think I got rid of the worst of it.
![K1_Sharina_N_TatComW_F](https://user-images.githubusercontent.com/46890653/57586005-726be900-7532-11e9-946d-6f470b6f7921.jpg)

**Comment by ebmar** [comment]
Yes! That is an amazing improvement ,**DP**! I had noticed about the issue for some time but taking the "_just change the heads reference_" path instead, hahah. Great to see an actual fix for the head; she looks fantastic right now.

**Comment by DarthParametric** [comment]
N_TatComW_F is the same in both. Is there a duplicate of it under another name?

**Comment by ebmar** [comment]
> N_TatComW_F is the same in both... There is Comm_a_F_W, but that's a reskin presumably.

Hahah, yeah - out of the game files' reach I mistakenly recognize `Comm_a_F_W` and/or `comm_w_F_M` as `N_TatComW_F`. My bad!

**Comment by DarthParametric** [comment]
They are certainly based on it, but they are clearly repaints. They also seem to lack any texture for the teeth. Maybe the TSL models use a different texture for that.

**Comment by DarthParametric** [comment]
This actually extends further, since naturally the head is reused with different textures.

* N_TatComW_F (original fix)
* comm_a_f (although eyelids and teeth use N_comm_a_F UVs)
* comm_a_f2 (straight retexture)

**Comment by DarthParametric** [comment]
Not quite as pronounced (although this is a poor shot) on the Asian female commoner head, but here's Elora on Manaan:
![Asian_Female_Commoner_Head](https://user-images.githubusercontent.com/46890653/57763389-9f8fe580-7744-11e9-9456-28dea6c94a38.jpg)

**Comment by DarthParametric** [comment]
Seems like N_TatComW_F got upscaled to 1024x1024 without further changes. Presumably the intention was to fix the mismatch in the skin tone, but that wasn't done.
